workspaces/libnpmpublish/test/publish.js

```diff
@@ -784,7 +784,7 @@ t.test('automatic provenance in unsupported environment', async t => {
   mockGlobals(t, {
     'process.env': {
       CI: false,
-      GITHUB_ACTIONS: false,
+      GITHUB_ACTIONS: undefined,
```

Comment:
Turns out that setting this to false still causes `ciInfo.name` to evaluate to "GITHUB_ACTIONS". To properly simulate NOT running in GitHub Actions, this needs to be undefined.

Reply:
ah yes, I guess it would be very surprising if this env var was set to anything on some other platform?